rest/model/distro.go

```diff
@@ -125,6 +126,10 @@ func (apiDistro *APIDistro) BuildFromService(h interface{}) error {
 	apiDistro.Setup = ToAPIString(d.Setup)
 	apiDistro.Teardown = ToAPIString(d.Teardown)
 	apiDistro.User = ToAPIString(d.User)
+	if d.BootstrapMethod == "" {
```

Comment:
I feel like the model package should just be for model conversions. Defaulting data like this should be elsewhere (maybe in the data layer)

Reply:
This is what is currently done to handle legacy/tunable planner versions (immediately below this) because the validation check is done right after this `BuildFromService` is called (`validateDistro()`). If I moved this out of here, then the check for legacy/non-legacy will be inconsistent with how the planner handles converting empty strings to the legacy value.